taskcluster/ac_taskgraph/loader/build_config.py

```diff
@@ -26,7 +26,7 @@ def loader(kind, path, config, params, loaded_tasks):
             }
         }
         for component in get_components()
-        for build_type in ('regular', 'release', 'snapshot')
+        for build_type in ('regular', 'release', 'snapshot', 'nightly-release')
```

Comment:
Nit: I think `nightly` is a better name than `nightly-release` for 2 reasons: 

1. It avoids reusing `release` which has too many different meanings
2. It's compliant with what we have on mozilla-central.

Reply:
Renamed to `nightly`. The task descriptions are clearer now 👍